### ISSUE TITLE
Add improved config fixture

### ISSUE THREAD

**Issue by schmengler** [tddwizard/magento2-fixtures#6]
The core config fixtures (`@magentoConfigFixture`) have unintuitive syntax and some pitfalls. For example you might set a global value but the imported test database overrides this value on store level. A config fixture like this would be useful:

```
ConfigFixture::setGlobal($path, $value);
```

where no matter what, this value is valid globally.

Additionally, for completeness:

```
ConfigFixture::setForStore($storeId, $path, $value);
```

**Comment by Alexie81** [comment]
This is very important !!! Thanks !😊

**Comment by schmengler** [comment]
@Alexie81 @GalileoWebagentur I finally implemented it, coming to version 1.0.0!
